supported-version/src/versions/magento-open-source/composite.json

```
@@ -125,32 +125,45 @@
         "release": "2024-04-09T00:00:00+0000",
         "eol": "2027-04-09T00:00:00+0000"
     },
+    "magento/project-community-edition:>=2.4.8 <2.4.9": {
+        "magento": "magento/project-community-edition:>=2.4.8 <2.4.9",
+        "php": 8.4,
+        "composer": "2.8.8",
+        "mysql": "mariadb:11.4",
+        "elasticsearch": "opensearch:2.19.1",
+        "rabbitmq": "rabbitmq:4.0-management",
+        "redis": "redis:7.2",
+        "varnish": "varnish:7.6",
+        "nginx": "nginx:1.26",
+        "os": "ubuntu-latest",
+        "release": "2025-04-08T00:00:00+0000",
+        "eol": "2028-04-09T00:00:00+0000"
+    },
     "magento/project-community-edition": {
-        "magento": "magento/project-community-edition:>=2.4.7 <2.4.8",
-        "php": 8.3,
-        "composer": "2.7.4",
-        "mysql": "mariadb:10.6",
-        "elasticsearch": "elasticsearch:8.11.4",
-        "rabbitmq": "rabbitmq:3.12-management",
+        "magento": "magento/project-community-edition:>=2.4.8 <2.4.9",
+        "php": 8.4,
+        "composer": "2.8.8",
+        "mysql": "mariadb:11.4",
+        "elasticsearch": "opensearch:2.19.1",
```

```suggestion
        "elasticsearch": "elasticsearch:8.17.4",
```

I would consider either:

- the above
- removing the elasticsearch key (as support for ES is dropped) and adding a new sibling key called opensearch.